mteb/tasks/Classification/ara/OnlineStoreReviewSentimentClassification.py

```diff
@@ -0,0 +1,48 @@
+from __future__ import annotations
+
+from mteb.abstasks import AbsTaskClassification
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+N_SAMPLES = 2048
+
+class OnlineStoreReviewSentimentClassification(AbsTaskClassification):
+    metadata = TaskMetadata(
+        name="OnlineStoreReviewSentimentClassification",
+        dataset={
+            "path": "Ruqiya/Arabic_Reviews_of_SHEIN",
+            "revision": "8ea114aa27b82a52373203830dc2f5b540b6fcac",
+        },
+        description="This dataset contains Arabic reviews of products from the SHEIN online store.",
+        reference="https://huggingface.co/datasets/Ruqiya/Arabic_Reviews_of_SHEIN",
+        type="Classification",
+        category="s2s",
+        eval_splits=["train"],
+        eval_langs=["ara-Arab"],
+        main_score="accuracy",
+        date=("2024-05-01", "2024-05-15"),
+        form=["written"],
+        domains=["Reviews"],
+        task_subtypes=["Sentiment/Hate speech"],
+        license="apache-2.0",
```

Comment:
Are we sure about this license?
Not sure if one can license reviews (of Amazon, Shein, etc) with a permissive license like this

Reply:
I meant the license that I added in Hugging Face dataset , I added Apache-2.0
What do you think ?